### PR TITLE
Add device BLPPSCtrl

### PR DESCRIPTION
**Pull request by xresende**
Add BLPPSCtrl device (beamline personal protection system control) to open beamlines (CAX)

### CODE REVIEW

**Review by fernandohds564 — APPROVED** [review]

File: siriuspy/siriuspy/devices/blctrl.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"""Beamline Control."""
+
+from .device import Device as _Device
+
+
+class BLCtrl(_Device):
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/blctrl.py:6]: maybe we could use a name that refers to PPS system, this name is a little bit generic

File: siriuspy/siriuspy/devices/blctrl.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+"""Beamline Control."""
+
+from .device import Device as _Device
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/blctrl.py:3]: shouldn't it be derivated from DeviceNC?

File: siriuspy/siriuspy/devices/blctrl.py (hunk 1 above)
Comment (xresende) [reply siriuspy/siriuspy/devices/blctrl.py:6]: I am not sure what else in the CAX will be useful and implemented in this device, that is why I left generic. Another option is to break into many classes.

File: siriuspy/siriuspy/devices/blctrl.py (hunk 2 above)
Comment (xresende) [reply siriuspy/siriuspy/devices/blctrl.py]: done

File: siriuspy/siriuspy/devices/blctrl.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/devices/blctrl.py]: I like this last approach. Or at least include the prefix `pps_` in the property names.

Comment (anacso17) [reply siriuspy/siriuspy/devices/blctrl.py]: I also prefer the last approach

Comment (xresende) [reply siriuspy/siriuspy/devices/blctrl.py]: class renamed to `BLPPSCtrl`

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

**Pull request merged**